talisker/postgresql.py

```diff
@@ -109,20 +110,28 @@ def _format_query(self, query):
         query = FILTERED if query is None else query
         return query
 
-    def _record(self, msg, query, duration):
+    def _record(self, msg, query, duration, timeout=None, cancelled=False):
         talisker.Context.track('sql', duration)
 
         if self.query_threshold >= 0 and duration > self.query_threshold:
             extra = collections.OrderedDict()
             extra['trailer'] = self._format_query(query)
             extra['duration_ms'] = duration
             extra['connection'] = get_safe_connection_string(self)
+            if timeout:
```

Comment:
I don't think timeout can be 0 here but I'd still be happier if this was `is not None`